test/unit/math/mix/fun/svd_U_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include <test/unit/math/test_ad.hpp>
+
+TEST(MathMixMatFun, svd_U) {
+  auto f = [](const auto& x) { return stan::math::svd_U(x); };
+
+  stan::test::ad_tolerances tols;
+  //tols.hessian_hessian_ = 1e-2;
+  //tols.hessian_fvar_hessian_ = 1e-2;
```

Comment:
These options were used to limit the tolerance of the finite difference tests. If the tests work without them, nothing to worry about.